requests/sessions.py

```diff
@@ -387,6 +387,13 @@ def __init__(self):
         self.stream = False
 
         #: SSL Verification default.
+        #: Defaults to `True`, requiring requests to verify the TLS certificate at the
+        #: remote end.
+        #: If verify is set to `False`, requests will accept any TLS certificate
+        #: presented by the server, and will ignore hostname mismatches and/or
+        #: expired certificates, which will make your application vulnerable to
+        #: man-in-the-middle (MitM) attacks.
+        #: Only set this to `False` for testing.
```

Comment:
There are (rarely) good cases for setting this to False. I'd remove this line as it's likely to cause a fire-storm for the already overtaxed maintainers of this project

Reply:
I would agree! The reason that I mentioned this was to get ahead of questions about "why is this configurable in the first place?" - which is a reasonable question for someone to ask about an option like this. I'm not sure I grok the 'firestorm' part, though.

Happy to elide if there's strong feelings about it, but "when would I set this to False" is a fair question for a newcomer to ask.

Reply:
There are people who jump on things like this as soon as they find it and then they post about it in places to get attention and trigger a pile-on. The project has had issues like this in the past (for example, when it kept it's own certificate bundle copy from Mozilla instead of keeping that in certifi) and so given how sensitive people can be around this particular flag and topic, I'd rather leave this as an uncontroversial item. I think the "Only" in this will be controversial enough that people will get spitting mad and try to rile others up as well

Reply:
That’s unfortunate, and hopefully we can point folks to prior art in other libraries/PLs. I have seen pile-ons the other way - e.g where the defaults are _unsafe_. But I digress.

To make sure I understand your ask concretely: you want me to remove the “Only set this...” sentence from the comment (only) - ?

Reply:
Or phrase it as `"You might consider settings to False during local development or testing."` as I think that's the idea you're conveying

Reply:
Done.